src/main/java/lt/vu/persistence/orm/entities/Address.java

```diff
@@ -0,0 +1,50 @@
+package lt.vu.persistence.orm.entities;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Objects;
+
+@Entity
+@NamedQueries({
+        @NamedQuery(name = "Address.findAll", query = "select e from Address as e")
+})
+@Getter
+@Setter
+@Table(name = "ADDRESSES")
+public class Address implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    private String city;
+
+    private String street;
+
+    private String nr;
+
+    @OneToOne
+    private User user;
+
+    @OneToOne
+    private UserInfo guest;
```

Comment:
We should decide on registered/guest user naming. I think guest fits quite well here.